python/gtsam/tests/test_Triangulation.py

```diff
@@ -76,5 +79,63 @@ def test_TriangulationExample(self):
         # triangulated_landmark = triangulatePoint3(poses,bundlerCal, measurements, rank_tol, optimize)
         # self.gtsamAssertEquals(landmark, triangulated_landmark,1e-9)
 
+    def test_distinct_Ks(self):
+        K1 = Cal3_S2(1500, 1200, 0, 640, 480)
+        camera1 = PinholeCameraCal3_S2(self.pose1, K1)
+
+        K2 = Cal3_S2(1600, 1300, 0, 650, 440)
+        camera2 = PinholeCameraCal3_S2(self.pose2, K2)
+
+        # landmark ~5 meters infront of camera
+        landmark = Point3(5, 0.5, 1.2)
+
+        # 1. Project two landmarks into two cameras and triangulate
+        z1 = camera1.project(landmark)
+        z2 = camera2.project(landmark)
+        # two cameras
+        measurements = Point2Vector()
+        cameras = CameraSetCal3_S2()
+
+        measurements.append(z1)
+        measurements.append(z2)
+        cameras.append(camera1)
+        cameras.append(camera2)
+
+        optimize = True
+        rank_tol = 1e-9
+
+        triangulated_landmark = triangulatePoint3(cameras, measurements, rank_tol, optimize)
+        self.gtsamAssertEquals(landmark, triangulated_landmark, 1e-2)
+
+    def test_distinct_Ks_Bundler(self):
+        K1 = Cal3Bundler(1500, 0, 0, 640, 480)
+        camera1 = PinholeCameraCal3Bundler(self.pose1, K1)
+
+        K2 = Cal3Bundler(1500, 0, 0, 640, 480)
+        camera2 = PinholeCameraCal3Bundler(self.pose2, K2)
+
+        # landmark ~5 meters infront of camera
+        landmark = Point3(5, 0.5, 1.2)
+
+        # 1. Project two landmarks into two cameras and triangulate
+        z1 = camera1.project(landmark)
+        z2 = camera2.project(landmark)
+        # two cameras
+        measurements = Point2Vector()
+        cameras = CameraSetCal3Bundler()
+
+        measurements.append(z1)
+        measurements.append(z2)
+        cameras.append(camera1)
+        cameras.append(camera2)
```

Comment:
It is good practice (it might also be my personal choice) to create a variable where it is used. That would mean 
```
        measurements = Point2Vector()
        measurements.append(z1)
        measurements.append(z2)

        cameras = CameraSetCal3Bundler()
        cameras.append(camera1)
        cameras.append(camera2)
```
Would you say this is more readable? It is up to you ultimately.

Reply:
nit: You could also create the cameras vector immediately after camera1 and camera2 were defined.